patcher.sh

```diff
@@ -156,28 +157,24 @@ patch_am2r ()
 			! [[ "${target}" = "${target,,}" ]] && mv "${target}" "${target,,}"
 		' \;
 
-		# GameMaker games (like AMR2) link to OpenSSL 1.0.0, which is outdated and insecure.
-		# When attempting to link to newer versions, an error is raised at runtime claiming it cannot find
-		# the outdated version of OpenSSL, even though it has been patched to link to the newer version.
-		# After replacing it with libcurl, versioning is ignored, and the binary starts just fine.
-
-		# Currently, patchelf has a bug where this does not work correctly
-		# So it will stay commented out until it does
-		#echo "Patching insecure OpenSSL dependency with libcurl..."
-		#patchelf "$GAMEDIR/runner" \
-		#	--replace-needed "libcrypto.so.1.0.0" "libcurl.so" \
-		#	--replace-needed "libssl.so.1.0.0" "libcurl.so"
-
-		# An environment variable needs to be set on Mesa to avoid a race related to multithreaded shader compilation.
-		# To do this, we move the original executable to a hidden file, and create a bash script with the needed variable in place of the original.
-		echo "Creating wrapper script to fix Mesa support..."
+		if [ "$PATCHOPENSSL" = true ]; then
+			# GameMaker games (like AMR2) link to OpenSSL 1.0.0, which is outdated and insecure.
+			# When attempting to link to newer versions, an error is raised at runtime claiming it cannot find
+			# the outdated version of OpenSSL, even though it has been patched to link to the newer version.
+			# After replacing it with libcurl, versioning is ignored, and the binary starts just fine.
+			echo "Patching insecure OpenSSL dependency with libcurl..."
```

Comment:
Reading through this, maybe the text should be changed to "deprecated" rather than "insecure".